halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/services/v1/GenerateService.java

```diff
@@ -129,11 +129,11 @@ public GenerateResult generateConfig(String deploymentName) {
     for (SpinnakerProfile profile : spinnakerProfiles) {
       path = defaultFileSystem.getPath(spinnakerOutputPath, profile.getProfileFileName());
       ProfileConfig config = profile.getFullConfig(deploymentName);
-      log.info("Writing " + profile.getProfileName() + " profile to " + path + " with " + config.getRequiredFiles().size() + " required files");
+      log.info("Writing " + profile.getArtifact().getName() + " profile to " + path + " with " + config.getRequiredFiles().size() + " required files");
```

Comment:
Could pull out the methods you're calling multiple times into variables within this block.